Integrations/EWSv2/req.txt

```diff
@@ -0,0 +1,28 @@
+asn1crypto==0.24.0

```

Comment:
what this is used for? you already have it in the PipFile

Reply:
It's not needed. File I generated during a test. Will remove.